contracts/ForkDelta.sol

```diff
@@ -37,46 +43,71 @@ contract ForkDelta {
     depositingTokenFlag = false;
   }
 
+  /// This default function prevents people sending Ether directly into the contract.
```

Comment:
I would change this to:
/// The fallback function. Ether transfered into the contract is not accepted.